ts/Series/Networkgraph/NetworkgraphSeries.ts

```diff
@@ -208,26 +206,34 @@ class NetworkgraphSeries extends Series {
      */
     public initDataLabels(this: NetworkgraphSeries): SVGElement {
         const series = this,
-            shouldAnimate = series.dlShouldAnimate;
+            dlOptions = series.options.dataLabels,
+            shouldAnimate = series.shouldAnimateDataLabels;
+
+        let fadeDuration = 500;
+        if (dlOptions &&
+            dlOptions.animation &&
+            dlOptions.animation.fadeDuration) {
+            fadeDuration = dlOptions.animation.fadeDuration;
+        }
```

Comment:
From v11 we can use ES6 syntax, so these 5 lines can be written as 

`const fadeDuration = dlOptions?.animation?.fadeDuration ?? 500;`